region/lowernorfair/west.json

```diff
@@ -112,34 +103,40 @@
           "to": [
             {
               "id": 3,
-              "requires": [
-                {"or": [
-                  "heatProof",
-                  "canHeatRun"
-                ]},
-                {"or": [
-                  {"and": [
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
                     "SpaceJump",
-                    {"heatFrames": 400}
-                  ]},
-                  {"and": [
+                    {"heatFrames": 200}
+                  ]
+                },
+                {
+                  "name": "Without Acid",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
                     "UsedAcidChozoStatue",
```

Comment:
`UsedAcidChozoStatue` implies having spacejump making this the same as the base strat.

Reply:
I tried not to make that assumption to prevent the logic from breaking if someone were to mod the statue.